frontend/packages/kubevirt-plugin/src/selectors/vm-template/combined-dependent.ts

```diff
@@ -45,12 +45,14 @@ export const getOperatingSystems = (templates: TemplateKind[], userTemplate: str
 
 export const getWorkloadProfiles = (
   templates: TemplateKind[],
-  { flavor, os, userTemplate }: { flavor: string; os: string; userTemplate: string },
+  { flavor, os, userTemplateName }: { flavor: string; os: string; userTemplateName: string },
```

Comment:
we should get just the userTemplate here in templates and not have to look it up retrospectivelly through common templates + user template

Reply:
repeating myself, but let's not pass the name but just the `[template]` for userTemplates